src/client.js

```diff
@@ -54,7 +56,7 @@ var client = function (context) {
             }
 
             return Promise.props({
-                current:  methods.current(params),
+                current: methods.current(params),
```

Comment:
Can you keep it aligned please ? Sorry, I should have added eslint rules for all mozaïk modules…